modules/ROOT/pages/clauses/index.adoc

```diff
@@ -169,7 +169,7 @@ These comprise sub-clauses that must operate as part of reading clauses.
 m| xref::clauses/where.adoc[WHERE]
 | Adds constraints to the patterns in a `MATCH` or `OPTIONAL MATCH` clause or filters the results of a `WITH` clause.
 
-m| xref::clauses/order-by.adoc[+ORDER BY [ASC[ENDING] \| DESC[ENDING]]+]
+m| xref::clauses/order-by.adoc[ORDER BY [ASC[ENDING] \| DESC[ENDING]]]
```

Comment:
This does not render properly 

<img width="314" alt="Screenshot 2023-05-29 at 11 53 10" src="https://github.com/neo4j/docs-cypher/assets/112686610/55f1ec0e-e9b6-419a-a2af-0b21ef2584ba">

Reply:
Ah, good catch! The brackets needed escaping (already before, but the issue was hidden by the extra `+`).